dialog-el.html

```diff
@@ -6,11 +6,19 @@
     :host {
       position: absolute;
       display: inline-block;
+      transform: translate(0,0);
```

Comment:
I made a lot of the changes based on what I saw in fs-dialog. It had a default transform value like this, but if we don't need it then I'm fine with taking this out.